R/scripts/hosp_run.R

```diff
@@ -167,6 +174,14 @@ if(run_age_adjust){
   # NOTE(jwills): this file would ideally live inside of the hospitalization package as an .Rdata object
   prob_dat <- readr::read_csv(paste(opt$p,"sample_data","geoid-params.csv",sep='/'))
 
+  # Check that all geoids are in geoid-params.csv
+  geodata <- report.generation:::load_geodata_file(file.path(config$spatial_setup$base_path, config$spatial_setup$geodata),5,'0',TRUE)
+  in_geoids <- geodata[[config$spatial_setup$nodenames]]
+  missing_geoids <- setdiff(in_geoids, prob_dat$geoid)
+  if(length(missing_geoids) > 0) {
+    warning(paste("There are geoids in geodata that are not present in geoid-params.csv:", missing_geoids))
```

Comment:
This should only be a warning if hospitalization$paths$run_age_adjust = TRUE in the config. Can we also add "The hospitalization outcomes will not be calculated for geoids not present in geoid-params.csv."?

Reply:
It's hard to see here, but it's in a if(run_age_adjust) block, so that it only runs when run_age_adjust = TRUE.

Updating to the more informative message